spog/ui/src/pages/search.rs

```diff
@@ -155,6 +155,48 @@ pub fn search(props: &SearchProperties) -> Html {
         },
     );
 
+    let onsort_advisory = {
```

Comment:
I think you could create your own hook for this. But it's also fine the way it's now!

Reply:
I've moved this piece of code to `use_unified_search` so it does not repeat multiple times